package.json

```diff
@@ -151,7 +151,8 @@
       "node_modules/*",
       "playwright.config.js",
       "tests/*",
-      "packages/*"
+      "packages/*",
+      "frontend/public/*"
```

Comment:
should we have front/public in the BE package.json?

Reply:
yeah I think it needs to be there so that it doesn't try and lint that folder if we run lint from here on the CI or something